packages/contracts-por/contracts/TrueCurrencyWithProofOfReserve.sol

```diff
@@ -39,7 +39,7 @@ abstract contract TrueCurrencyWithProofOfReserve is TrueCurrency, IProofOfReserv
         uint256 reserves = uint256(signedReserves);
 
         // Sanity check: is chainlink answer updatedAt in the past
-        require(block.timestamp >= updatedAt);
+        require(block.timestamp >= updatedAt, "TrueCurrency: invalid PoR updatedAt");
```

Comment:
We're missing a test for this require statement.

Reply:
Added test.